package.json

```diff
@@ -25,6 +25,7 @@
     "istanbul-middleware": "0.2.0",
     "load-script": "0.0.5",
     "lodash": "2.4.1",
+    "mocha": "~2.2.5",
```

Comment:
I think you could use it with browserify (cf. `mocha/bin/_mocha`).
However, with your current implementation, this is an error of mine.
Will put it back to `devDependencies`.